pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -26,6 +26,12 @@ self: super: {
   # successfully with recent versions of the compiler).
   bin-package-db = null;
 
+  # release is taking long
+  c2hsc = appendPatch super.c2hsc (pkgs.fetchpatch {
+    url = "https://github.com/jwiegley/c2hsc/commit/490ecab202e0de7fc995eedf744ad3cb408b53cc.patch";
+    sha256 = "sha256-B0OcZkVrLTBCCBJL+b4vfxSTBDfTT5lKYOie3Pe187A=";
```

Comment:
A week or so ago, I think there was a PR removing all the SRI hashes from nixpkgs.  Could you replace this with a normal hash?

(Or maybe I am remembering incorrectly, and these SRI hashes are actually okay)

Reply:
I think that;s now fine https://github.com/NixOS/nixpkgs/pull/89308#issuecomment-638795785, but I can change it if I misunderstood.

Reply:
I want to merge this in, but I don't understand if this hash type is allowed.

This comment appears to say that `sha256 = <srihash>` should not be allowed:

https://github.com/NixOS/nixpkgs/issues/89275#issuecomment-636874367

Reply:
done